admin/controllers/pages/views/save/model.php

```diff
@@ -56,10 +57,9 @@
 		$data = array( $page->id, $_POST['override_positions'][$n], $_POST['override_positions_widgets'][$n], $page->id, $_POST['override_positions'][$n], $_POST['override_positions_widgets'][$n] );
 		$override_success = DB::exec("insert into page_widget_overrides (page_id, position, widgets) values (?,?,?) on duplicate key update page_id=?, position=?, widgets=?", $data);
 	}
-	
-	
 
-	CMS::Instance()->queue_message('Page created/updated','success',Config::uripath().'/admin/pages');
+	$msg = 'Page <a href="' . Config::uripath() . '/admin/pages/edit/' . $page->id . '/' . $page->content_type . '/' . $page->view . '">' . $page->title . '</a> ' . $status . ($new_page ? 'created' : 'updated');
```

Comment:
templating?

Reply:
fixed in b772c9972b1b6528cd219293d837976ffad29f09